sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -15,108 +15,108 @@ public partial class CloudMachine
         public void Save(string filepath) { }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct Value
+    public readonly partial struct Variant
```

Comment:
Yay!   We're bringing back VB6 a bit at time.   Let's do `On Error Resume Next` next!